app/build.gradle

```diff
@@ -71,47 +71,55 @@ android {
 }
 
 dependencies {
+    //Android Support
     compile 'com.android.support:support-v4:21.0.3'
-    compile 'com.viewpagerindicator:library:2.4.1@aar'
-    compile 'com.google.code.gson:gson:2.3'
-    compile 'org.jsoup:jsoup:1.7.3'
-    compile 'joda-time:joda-time:2.3'
-    compile 'com.jakewharton.timber:timber:2.3.0'
-    compile 'com.github.chrisbanes.actionbarpulltorefresh:library:0.9.9'
-    compile ('com.github.chrisbanes.actionbarpulltorefresh:extra-abc:0.9.9'){
-        exclude group: 'com.android.support', module:'support-v4'
-        exclude group: 'com.android.support', module:'appcompat-v7'
-    }
-    compile ('de.keyboardsurfer.android.widget:crouton:1.8.4') {
-        exclude module: 'support-v4'
-    }
-    compile 'com.google:volley:4.3'
-    compile 'com.google.android.analytics:analytics:3'
-    compile 'com.jakewharton:disklrucache:2.0.2'
-    compile 'com.google.android:youtube-android-player-api:1.0.0'
+    compile 'com.android.support:appcompat-v7:21.0.3'
+    compile 'com.android.support:cardview-v7:21.0.3'
+    compile 'com.android.support:mediarouter-v7:21.0.3'
 
+    //Play Services
     compile 'com.google.android.gms:play-services-location:6.5.87'
     compile 'com.google.android.gms:play-services-games:6.5.87'
     compile 'com.google.android.gms:play-services-identity:6.5.87'
     compile 'com.google.android.gms:play-services-plus:6.5.87'
     compile 'com.google.android.gms:play-services-appstate:6.5.87'
-    
+
+    //Google
+    compile 'com.google.code.gson:gson:2.3'
+    compile 'com.google:volley:4.3'
+    compile 'com.google.android:youtube-android-player-api:1.0.0'
     compile 'com.google.http-client:google-http-client-gson:1.18.0-rc'
     compile 'com.google.apis:google-api-services-plus:v1-rev126-1.18.0-rc'
     compile 'com.google.apis:google-api-services-youtube:v3-rev106-1.18.0-rc'
     compile 'com.google.api-client:google-api-client-android:1.18.0-rc'
-    compile 'com.android.support:appcompat-v7:21.0.3'
-    compile 'com.android.support:cardview-v7:21.0.3'
-    compile 'com.android.support:mediarouter-v7:21.0.3'
     compile 'com.google.android.apps.dashclock:dashclock-api:2.0.0'
-    compile 'com.squareup:otto:1.3.4'
+    compile 'com.google.zxing:android-integration:3.1.0'
+    compile 'com.google.zxing:core:3.1.0'
+
+    //Square
+    compile 'com.squareup:otto:1.3.6'
     compile 'com.squareup.okhttp:okhttp:1.6.0'
```

Comment:
I think it introduces major changes. Before 2.x, it was used just like HTTPUrlConnection but there were magic underneath. But after 2.x, API changed.